app/editor/src/features/content/list-view/utils/getFilter.ts

```diff
@@ -33,14 +33,9 @@ export const getFilter = (
     includeHidden: filter.includeHidden ? true : undefined,
     onlyHidden: filter.onlyHidden ? true : undefined,
     onlyPublished: filter.onlyPublished ? true : undefined,
-    startDate: filter.startDate
-      ? moment(filter.startDate).toISOString()
-      : setTimeFrame(filter.timeFrame as number)?.toISOString(),
-    endDate: filter.endDate
-      ? moment(filter.endDate).toISOString()
-      : filter.startDate
-      ? moment(filter.startDate).endOf('day').toISOString()
-      : undefined,
+    startDate: filter.startDate ? moment(filter.startDate).toISOString() : undefined,
+    endDate: filter.endDate ? moment(filter.endDate).toISOString() : undefined,
+    dateOffset: filter.timeFrame ? +filter.timeFrame : undefined,
```

Comment:
`dateOffset` is different than the `startDate` and `endDate` filters.  It was removed for some reason.